package.json

```diff
@@ -4,8 +4,7 @@
   "description": "Finite State Machines and Statecharts for the Modern Web.",
   "workspaces": {
     "packages": [
-      "packages/*",
-      "packages/xstate-viz/example"
+      "packages/*"
```

Comment:
We should also remove this from here:
https://github.com/statelyai/xstate/blob/6245be838581e2cf84ac580303262e46b6cebd56/.changeset/config.json#L14-L15